backend/scripts/seed_db.sh

```
@@ -41,6 +45,38 @@ parse_arguments() {
   fi
 }
 
+create_mock_secrets() {
+  local app_dir=$1
+  local secrets_file="${app_dir}.app.secrets.json"
+  local app_json="${app_dir}app.json"
+  local app_name=$(basename "$app_dir")
```

Shellcheck (0.10.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)